docs/api/embedding.rst

```diff
@@ -24,10 +24,6 @@ and utility for intrinsic evaluation of text embeddings.
 API Reference
 -------------
 
-.. raw:: html
-
-   <script type="text/javascript" src='../../_static/js/auto_module_index.js'></script>
-
 .. automodule:: gluonnlp.embedding
     :members: register, create, list_sources
```

Comment:
can we use the automodule to list everything inside gluonnlp.embedding too? This way we won't have to worry about listing new classes in API reference

Reply:
same for the API doc for any other module.

Reply:
I updated the other API references accordingly